Cargo.toml

```diff
@@ -69,6 +70,7 @@ exclude = ["tock"]
 members = [
     "apis/interface/buttons",
     "apis/interface/buzzer",
+    "apis/interface/servo",
```

Comment:
Similarly, move this between `apis/interface/leds` and `apis/kernel/low_level_debug` to put it into alphabetical order.